### ISSUE TITLE
Language selector drop-down reloads the page at each change when operated with the keyboard

### ISSUE THREAD

**Issue by davidbenque** [cryptpad/cryptpad#1602]
### Description

On the instance home-page (and other static pages) footer, one cannot browse through the available options with ⬆️ ⬇️ arrow keys without actually triggering the change and a page reload at each change. 

### Steps to reproduce

1. On the instance home-page tab to the language selector in the footer
2. Use arrow keys to select another option
3. Observe that the page reloads and the language change is applied after you pressed one arrow

### Expected behavior

Use arrow keys to browse through options and enter to submit. (and match drop-down behavior elsewhere in CryptPad)

### Which operating system are you using?

Linux/BSD/UNIX

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

uBlock Origin, Bitwarden

### Version

2024.6.1

**Comment by dariiing** [comment]
Hi! You can actually navigate through the options without reloading the page by pressing the Space bar to open the dropdown. From there, the user can select their preferred language. This is the standard functionality of the `select` attribute. The concern you mentioned shouldn’t really be a problem, since ultimately, the language is selected and the page reloads accordingly if the user chooses not to open the dropdown, but go through the options one by one.

**Comment by davidbenque** [comment]
Removing this issue from projects since it is standard behavior for the select element. Will keep this issue open for now.
